scripts/benchmark.sh

```diff
@@ -0,0 +1,220 @@
+#!/bin/bash
+WORKDIR=$(pwd)
+OUTPUT_DIR="${WORKDIR}/out"
+SERVERS=("172.20.0.2" "172.20.0.3" "172.20.0.4" "172.20.0.5")
+CLUSTER_PEERS=(
+    ""
+    "${SERVERS[2]}:2379 ${SERVERS[3]}:2379"
+    "${SERVERS[1]}:2379 ${SERVERS[3]}:2379"
+    "${SERVERS[1]}:2379 ${SERVERS[2]}:2379"
+)
+# container endpoints
+XLINE_TESTCASE=(
+    "node1  false ${SERVERS[1]}:2379"
+    "node2  false ${SERVERS[2]}:2379"
+    "client false ${SERVERS[3]}:2379"
+    "client true  ${SERVERS[1]}:2379 ${SERVERS[2]}:2379 ${SERVERS[3]}:2379"
+)
+ETCD_TESTCASE=(
+    "node1  false ${SERVERS[1]}:2379"
+    "node2  false ${SERVERS[2]}:2379"
+    "client false ${SERVERS[3]}:2379"
+)
+KEY_SPACE_SIZE=("1" "100000")
+CLIENTS_TOTAL=("1 50" "10 300" "50 1000" "100 3000" "200 5000")
+FORMAT="%-8s\t%-12s\t%-8s\t%-s\n"
+
+# generate benchmark command by arguments
```

Comment:
The name is usually obvious, but the arguments are not. Please add comment meaning for the function.

Reply:
done

Reply:
We usually put the argument comments above the method definition, as you can't always make sure to read the argument at the beginning of the function definition. Here's the example:

```rust
# function doc
# args:
#    1. xxxx
#    2. xxxx
```